src/App.svelte

```diff
@@ -89,20 +95,26 @@
 </main>
 <footer>
   <div class="options">
-    <label>
+    <label class="big-only">
       <input type="checkbox" bind:checked={$onlyMovingo} id="toggleMovingo" />
       Movingo
     </label>
     <label>
-      From
+      <span class="big-only">From</span>
       <input type="datetime-local" step="1" bind:value={fromTime} />
     </label>
   </div>
   <div class="buttons">
-    <button on:click={toggleLocation} disabled={loading}>
+    <button on:click={toggleLocation} disabled={loading} class="big-only">
       Change direction
     </button>
-    <button on:click={onClick} disabled={loading} class="refresh">
+    <button on:click={toggleLocation} disabled={loading} class="icon smol-only">
+      <ChangeDirectionIcon />
+    </button>
+    <button on:click={toggleMovingo} disabled={loading} class="icon smol-only" class:depressed={$onlyMovingo}>
```

Comment:
`class:depressed={$onlyMovingo}>` this only really works when I simulate an iPhone through Safari's responsive design mode. On my real phone it's a bit wonky (stays bright after toggle, but then goes dark on tapping the refresh button...)

Reply:
A button is not enough to represent a toggled state. If we want to have a button for this maybe if we had two icons, one with a line (indicating crossed over) and one regular. And then switch the icon when toggling.

Reply:
👍 I added a second icon. It feels like the toggling behavior is still simple enough to not extract the button into some generic component, do you agree?

Reply:
I agree, looks good